testpilot/frontend/static-src/app/views/experiment-page.js

```diff
@@ -217,6 +248,11 @@ export default PageView.extend({
     return this;
   },
 
+  beforeRender() {
+    this.activeUser = app.me.hasAddon;
+    this.isFirefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
```

Comment:
I copy/pasted this from [the landing page's view](https://github.com/mozilla/testpilot/blob/35802081bee163996b6766d8d119de769284e321/testpilot/frontend/static-src/app/views/landing-page.js#L21); worth improving, but probably orthogonal to this PR. I'll file a bug.